src/svc_agt.c

```diff
@@ -43,48 +40,46 @@ int svc_agt_init (const char *svcagt_directory)
 {
 	int err;
 	struct stat stat_buf;
-
+	svcagt_log (LEVEL_INFO,  "processing svc_agt_init..\n");
 	if (NULL == svcagt_directory) {
-		dbg ("svc_agt init NULL svcagt directory provided\n");
+		svcagt_log (LEVEL_DEBUG,"svc_agt init NULL svcagt directory provided\n");
```

Comment:
But if the logger has not been set up yet, then obviously this call won't succeed.
That's what dbg is for.  See conversation